templates/client.ex.eex

```diff
@@ -0,0 +1,16 @@
+defmodule <%= prj_name %>.Daemon do
```

Comment:
The file is called client but the definition is creating a deamon. Any reason for the missmatch?